<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.1m"
		durationms="67095"
		type="root"
		rid="169ba946fc52cdf8"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="I-Ray"
		rxversion="10.5.1.0"
		host="IRAYSERVER"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1600x900"
		timestamp="10/19/2023 11:51:59 AM"
		timestampiso="2023-10-19T06:21:59"
		endtime="10/19/2023 11:53:06 AM"
		timeoutfactor="1"
		runid=""
		sutversion=""
		module="Fittingvalues">
		<detail>
			Fittingvalues
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="01:07.093"
			timeRelativeToTestModuleStartTime="01:07.093"
			timeWallClock="11:53:06"
			time="01:07.093"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
